text/metrics/0001-naming-conventions.md

```diff
@@ -0,0 +1,149 @@
+# Metric naming conventions
+
+## Purpose
+
+Metric names and labels are the primary read-interface to metric data. The names and taxonomy need to be understandable and discoverable during routine exploration -- and this becomes critical during incidents.
+
+## Guidelines
+
+Namespace similar metrics together. Define top-level namespaces for common metric categories: for OS, like CPU and network; for app runtimes, like the JVM. This aids in discovery and adhoc comparison
+
+Provide consistent names for common label. Provides discoverability and disambiguation similar to metric names.
+
+Avoid semantic ambiguity. Use namespaced metric names in cases where similar metrics have significantly different implementations across the holistics of metrics. For example, every garbage collected runtime has a slightly different strategies and measures. Using common metric names for GC, not namespaces by the runtime, could create dissimilar comparisons and confusion for end users. Measures of operating system memory are similar.
+
+## Conventions
+
+All metrics have a limited set of common labels:
+* `system.host`
+
+### Operation System
```

Comment:
We should be careful to distinguish OS-specific and OS-agnostic metrics and labels. For example, memory metrics might have different label values depending on OS - Linux will have a different set of types of memory (private, slab, etc.) than Windows. So maybe we could add a generic statement - "standard metric names and labels should be OS-agnostic".

Reply:
Removed specific metric naming guidance from this doc -- this will be handled in a future otep.